easybuild/easyconfigs/i/IntelClusterChecker/IntelClusterChecker-2017.1.016.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "IntelBase"
+
+name = 'IntelClusterChecker'
+version = '2017.1.016'
+
+homepage = 'https://software.intel.com/en-us/intel-cluster-checker'
+description = """Verifies cluster components work together ― for 
+ - better uptime and productivity and lower total
+ - cost of ownership (TCO)
+ """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_clck_p_%(version)s.tgz']
+
+checksums = ['84e7c00bf1f9b87e248dd62e4df7e3b7']
```

Comment:
@shahzebsiddiqui Do you mind making this a `SHA256` checksum? Also, no need for an empty line below `sources = ...`, makes sense to group these together.